products/accesscontextmanager/api.yaml

```diff
@@ -260,3 +260,154 @@ objects:
+          be in zero or more perimeter bridges. A perimeter bridge only contains
+          resources. Cross project operations are permitted if all effected
+          resources share some perimeter (whether bridge or regular). Perimeter
+          Bridge does not contain access levels or services: those are governed
+          entirely by the regular perimeter that resource is in.
+
+          Perimeter Bridges are typically useful when building more complex
+          toplogies with many independent perimeters that need to share some data
+          with a common perimeter, but should not be able to share data among
+          themselves.
+        values:
+          - :PERIMETER_TYPE_REGULAR
+          - :PERIMETER_TYPE_BRIDGE
+        default_value: :PERIMETER_TYPE_REGULAR
+      - !ruby/object:Api::Type::NestedObject
+        name: 'status'
+        description: |
+          ServicePerimeter configuration. Specifies sets of resources,
+          restricted/unrestricted services and access levels that determine
+          perimeter content and boundaries.
+        properties:
+          - !ruby/object:Api::Type::Array
+            name: 'resources'
+            description: |
+              A list of GCP resources that are inside of the service perimeter.
+              Currently only projects are allowed.
+              Format: projects/{project_number}
+            item_type: Api::Type::String
+          - !ruby/object:Api::Type::Array
+            name: 'accessLevels'
+            description: |
+              A list of AccessLevel resource names that allow resources within
+              the ServicePerimeter to be accessed from the internet.
+              AccessLevels listed must be in the same policy as this
+              ServicePerimeter. Referencing a nonexistent AccessLevel is a
+              syntax error. If no AccessLevel names are listed, resources within
+              the perimeter can only be accessed via GCP calls with request
+              origins within the perimeter. For Service Perimeter Bridge, must
+              be empty.
+
+              Format: accessPolicies/{policy_id}/accessLevels/{access_level_name}
+            item_type: Api::Type::String
+          - !ruby/object:Api::Type::Array
+            name: 'unrestrictedServices'
+            description: |
+              GCP services that are not subject to the Service Perimeter
+              restrictions. May contain a list of services or a single wildcard
+              "*". For example, if logging.googleapis.com is unrestricted, users
+              can access logs inside the perimeter as if the perimeter doesn't
+              exist, and it also means VMs inside the perimeter can access logs
+              outside the perimeter.
+
+              The wildcard means that unless explicitly specified by
+              "restrictedServices" list, any service is treated as unrestricted.
+              One of the fields "restrictedServices", "unrestrictedServices"
```

Comment:
Do we usually try to add validation for things like this or just let the api error out since it's pretty clearly documented?

Reply:
Just let it error out - we have no good means of validating multiple-field rules